packages/svelte-ux/src/lib/components/DateField.svelte

```diff
@@ -21,6 +22,13 @@
   $: actualFormat = format ?? $format_ux.settings.formats.dates.baseParsing ?? 'MM/dd/yyyy';
   $: actualMask = mask ?? actualFormat.toLowerCase();
 
+  type FieldClasses = NonNullable<ComponentProps<Field>['classes']>;
```

Comment:
Is `NonNullable` doing anything here?

Reply:
`NonNullable` was only being used to more easily access the optimistic type for `root`. I considered we might just want `string` but wasn't sure if the type should be derived.

I'll replace this.

Reply:
Resolved in [209b2c3](https://github.com/techniq/svelte-ux/pull/244/commits/209b2c393bdfd142f1b5e902f9b618350ffaf7bb)